packages/cozy-search/src/components/Search/SearchBarDesktop.jsx

```diff
@@ -114,23 +95,17 @@ const SearchBarDesktop = ({
           componentsProps={{
             inputBase: {
               onKeyDown: handleKeyDown,
-              endAdornment: isAssistantEnabled() ? (
-                <AssistantButton onClick={handleClick} size={size} />
-              ) : undefined
+              endAdornment: isAssistantEnabled() && (
```

Comment:
no warning because replacing undefined by false (if I'm right)?

Reply:
After verification no warning, but good idea to check 👍